src/utils/configValidators/required.js

```diff
@@ -10,7 +10,7 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-export default (config, key, currentValue) => {
+export default (key, currentValue) => {
   let err = "";
   if (currentValue == null) {
```

Comment:
I'd like to see this as `if (currentValue === undefined)`. I think validation should fail f the user passes in `null` for a required option.

Reply:
OMG...I'm not thinking straight. Ignore this comment too.